lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -172,45 +184,48 @@ define((require, exports, module) ->
       ]
       LABEL_PARAMETER_INDEX: 2
 
-      getWholeEquation: (range) ->
+      getMacrosArgumentRange: (session, argumentStartPos) ->
+        argumentRange = findSurroundingBrackets(session, argumentStartPos)
+        if argumentRange.mismatch
+          return null
+        else
+          return new Range(argumentRange.start.row, argumentRange.start.column + 1,
```

Comment:
The confusing part is that end column is not decremented. Is it because `findSurroundingBrackets` returns closed interval [start, end] while Range takes semi-opened [start, end) ?